salt/modules/pkg_resource.py

```diff
@@ -115,18 +115,26 @@ def parse_targets(name=None,
     if __grains__['os'] == 'MacOS' and sources:
         log.warning('Parameter "sources" ignored on MacOS hosts.')
 
+    version = kwargs.get('version')
+
     if pkgs and sources:
         log.error('Only one of "pkgs" and "sources" can be used.')
         return None, None
 
     elif pkgs:
+        if version is not None:
+            log.warning('\'version\' argument will be ignored for multiple '
```

Comment:
Since this is just a style question and not a functional issue, I am going to merge this and you two can discuss this later on. :]